src/Numerics/Mesh/Topologies.jl

```diff
@@ -189,7 +225,13 @@ struct BoxElementTopology{dim, T, nb} <: AbstractTopology{dim, T, nb}
         nabrtosend,
         origsendorder,
         bndytoelem,
-        bndytoface,
+        bndytoface;
+        elemtouvert = nothing,
```

Comment:
It's not ideal, but this could work for now.
```suggestion
        device_array = ClimateMachine.array_type(),
        elemtouvert = nothing,
```

Reply:
revised.